### ISSUE TITLE
/nick showing your nicked name instead of your own

### ISSUE THREAD

**Issue by 2000thehecf1** [OvercastNetwork/Issues#2344]
It showed your own nick crossed out in tab and it and your nicked one while writing on chat but know it only shows your nicked one in dms, when placing wools, etc. Not sure if intended but I don't think so.

**Comment by AnthonyTag** [comment]
Could you give an example with pictures? I'm not really following.

**Comment by 2000thehecf1** [comment]
@AnthonyTag https://gyazo.com/8c011a3dc3710bab8643a76b16efd34f https://gyazo.com/58a09dd756a702030da8a9402e44437a That's how it looked before, now it just shows the nicked one, and even when you die / complete an objective you appear with the nicked name.

**Comment by ghost** [comment]
![](http://i.imgur.com/yekfPQf.png?1)
![](http://i.imgur.com/HJqpxxG.png?1)

Like this?

This is what I get when my friends are nicked and on the same server. Works fine when they switch.

**Comment by 2000thehecf1** [comment]
@elephantLocator Yes it works fine for friends but not for the guy with the nick.

**Comment by ghost** [comment]
@2000thehecf1 Wait shouldn't it show the alt name for me, next to the strikethrough? It sometimes does: punishment messages etc. but not in server list

**Comment by 2000thehecf1** [comment]
@elephantLocator Yep, true.

**Comment by 2000thehecf1** [comment]
Also, looking at this pic: https://camo.githubusercontent.com/e6d29f652715a6cde64033c27aa7904a075c370e/687474703a2f2f692e696d6775722e636f6d2f79656b665051662e706e673f31

It seems like nicked players appear at the end of tab list (look at aboudaga on Blue Team) making them more recognizable.
